README.md

```diff
@@ -73,6 +74,7 @@ Read the [contribution guidelines](CONTRIBUTING.md) before submitting a resource
 ### Type System
 * Official [Type System npm package]
 * Official [Type System Rust crate] from which the npm package is compiled
+* [Graph types]: A Python library to parse ontology schemas and validate entities based on the generated Pydantic model
```

Comment:
I might rename this `Python Validator`, rather than `Graph Types`? It seems more descriptive/appropriate.

Reply:
The issue us that I don’t really know what things we might want to add to the library. Currently it’s just containing the validation for entities as we need that RN. Also, I would call it the same name here as the package itself.